packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -263,90 +266,117 @@ export default function NewCommentModal({
+          <article
+            className={classNames(
+              'flex flex-col items-stretch',
+              commentBoxClassNames,
+            )}
+          >
+            <header className="flex items-center mb-2">
+              <RoundedImage
+                imgSrc={authorImage}
+                imgAlt={`${authorName}'s profile`}
+                background="var(--theme-background-secondary)"
+              />
+              <div className="flex flex-col ml-2">
+                <div className="truncate typo-callout">{authorName}</div>
+                <time
+                  dateTime={publishDate.toString()}
+                  className="text-theme-label-tertiary typo-callout"
+                >
+                  {commentDateFormat(publishDate)}
+                </time>
+              </div>
+            </header>
+            <Markdown content={contentHtml} />
+          </article>
+          <div className="flex items-center px-2 h-11">
+            <div className="ml-3 w-px h-full bg-theme-divider-tertiary" />
+            <div className="ml-6 text-theme-label-secondary typo-caption1">
+              Reply to{' '}
+              <strong className="font-bold text-theme-label-primary">
+                {authorName}
+              </strong>
+            </div>
+          </div>
+          <div className="flex px-2">
+            <ProfilePicture user={user} size="small" />
+            <div
+              className={classNames(
+                'ml-3 flex-1 text-theme-label-primary bg-none border-none caret-theme-label-link whitespace-pre-line break-words break-words-overflow typo-subhead',
+                styles.textarea,
+              )}
+              ref={commentRef}
+              contentEditable
+              role="textbox"
+              aria-placeholder="Write your comment..."
+              aria-multiline
+              onInput={onInput}
+              onKeyDown={onKeyDown}
+              onPaste={onPaste}
+              tabIndex={0}
+              aria-label="New comment box"
+            />
           </div>
-        </header>
-        <Markdown content={contentHtml} />
-      </article>
-      <div className="flex items-center px-2 h-11">
-        <div className="ml-3 w-px h-full bg-theme-divider-tertiary" />
-        <div className="ml-6 text-theme-label-secondary typo-caption1">
-          Reply to{' '}
-          <strong className="font-bold text-theme-label-primary">
-            {authorName}
-          </strong>
-        </div>
-      </div>
-      <div className="flex px-2">
-        <ProfilePicture user={user} size="small" />
-        <div
-          className={classNames(
-            'ml-3 flex-1 text-theme-label-primary bg-none border-none caret-theme-label-link whitespace-pre-line break-words break-words-overflow typo-subhead',
-            styles.textarea,
+          <div
+            className="my-2 mx-3 text-theme-status-error typo-caption1"
+            style={{ minHeight: '1rem' }}
```

Comment:
Can't we add a class for this somehow?

Reply:
I think we have been moving away from CSS modules right? Probably the reason why the `1rem` was left there since then. Unfortunately, there could be some bug in the arbitrary value of tailwindcss and it results to be not working. What do you think an option we can take other than this?

Reply:
Hmm we really need to get this tailwind to work again, don't worry if that's blocking this.
We'll have to refactor quite a few things for that.

Reply:
Think I should create a ticket for this so once we get tailwind working again, we will migrate everything at once. What do you think?